docs/windows.md

```diff
@@ -0,0 +1,187 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports to run on Windows worker Node, and setups an overlay network
+which uses VXLAN tunneling to route packets between nodes.
+
+One HNS Network with [Transparent mode](https://docs.microsoft.com/en-us/virtualization/windowscontainers/container-networking/network-drivers-topologies)
+is created on a Windows worker Node, on which Open vSwitch Extension is enabled.
+The HNS Network uses the network adapter which is used to join the Windows Node
+into the cluster. An OVS bridge is created, and leverages the network adapter as
+the uplink interface. The original IP and MAC address of the network adapter is
+moved to the OVS bridge. 
+
+One HNS Endpoint is created on this HNS Network and attached on all containers
+in the Pod when the CNI ADD command is received. An internal OVS port is created
+at the same time, which has the same name as the HNS Endpoint. OpenFlow entries
+are installed on the OVS bridge to provide container networking.
+
+Each Windows Node is assigned  with a single subnet, and 
+[host-local IPAM plugin](https://github.com/containernetworking/plugins/tree/master/plugins/ipam/host-local)
+is invoked to allocate IPs from the subnet to all local Pods.
+
+### Components that run on Windows
+
+The following components should be configured and run on the Windows Node.
+* [kubernetes components](https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-nodes/)
+* OVS daemons
+* Antrea Agent
+* Kube-proxy
+
+Both Antrea Agent and OVS daemons support to run as process or Windows service.
+
+### Traffic walk
+* ***Intra-node traffic*** and ***Inter-node traffic*** On Windows Node, the
+forwarding path for packets that are between two local Pods or to a Pod on
+another Node is the same as Linux Node. Please refer to the [architecture](/docs/architecture.md).
+
+* ***Pod to external traffic*** Antrea Agent creates OpenFlow rules to perform
+SNAT on the packets sent from local Pod to an external IP or the Nodes' network,
+and the source IP will be rewritten to the Node's IP. Then the packets are
+forwarded to the OVS bridge. IP-Forwarding is enabled on the OVS bridge, and the
```

Comment:
maybe a stupid question, but what does "Then the packets are forwarded to the OVS bridge" mean here? Aren't the packets already "in" the OVS bridge after SNAT is performed?

Reply:
I just copy the description from Wenying's previous patch.

Actually the pod-->external traffic go through the following path:
localPod --> OVS --> host network --> OVS --> external

It's little complicated, I will remove `Then the packets are forwarded to the OVS bridge` because it's trully hard to be understood. User can found the details in design doc. This doc is mainly for brief summary and installation instructions.